src/frontend/js/components/ReferralDetail/TabMessages.tsx

```diff
@@ -508,6 +514,7 @@ export const TabMessages = ({ referral }: TabMessagesProps) => {
                 </ul>
               ) : null}
             </div>
+            <FormattedMessage {...messages.helpText} />
```

Comment:
I think this would be nicer in darkgray, centered, and with a little margin on top